client/payment-details/timeline/test/map-events.js

```diff
@@ -156,6 +179,28 @@ describe( 'mapTimelineEvents', () => {
 							percentage: 0.0195,
 							fixed: 15,
 							fixed_currency: 'USD',
+							history: [
+								{
+									type: 'base',
+									percentage_rate: 0.014,
+									fixed_rate: 20,
+									currency: 'gbp',
+								},
+								{
+									type: 'additional',
+									additional_type: 'international',
+									percentage_rate: 0.014999999999999998,
```

Comment:
This rate `0.014999999999999998` looks strange to me. Does our server really respond with this `percentage_rate`?  Or do you have any reason to mock up this value?